README.md

```diff
@@ -168,4 +168,35 @@ Follow these steps once to setup CI/CD pipeline with [`Travis CI`](https://travi
  ```
  ### One Time Travis Setup
   * All notifications are sent to Slack channel. Reconfigure `.travis.yml` file to send notification to your own channel(s) if desired.
-  * Failure notifications are sent to email addresses configured in the `.travis.yml` file. Change as desired.
\ No newline at end of file
+  * Failure notifications are sent to email addresses configured in the `.travis.yml` file. Change as desired.
+  * Encrypt these ENV variables in the travis pipeline file. This is one off setup:
+  ```bash
+      heroku config:set `cat .env | grep SECRET_KEY` -a khalidashraf-todo-app
+      heroku config:set `cat .env | grep TRELLO_KEY` -a khalidashraf-todo-app
+      heroku config:set `cat .env | grep TRELLO_TOKEN` -a khalidashraf-todo-app
+      heroku config:set `cat .env | grep BOARD_ID` -a khalidashraf-todo-app   
+  ```
+  * Encrypt DOCKER_PASS env variable in the travis pipeline file. This is one off setup:
+  ```bash
+      heroku config:set `cat .env | grep SECRET_KEY` -a khalidashraf-todo-app
+      heroku config:set `cat .env | grep TRELLO_KEY` -a khalidashraf-todo-app
+      heroku config:set `cat .env | grep TRELLO_TOKEN` -a khalidashraf-todo-app
+      heroku config:set `cat .env | grep BOARD_ID` -a khalidashraf-todo-app   
```

Comment:
These commands look wrong - they're copies of the snippet below for setting up Heroku.

Did you mean to put some example `travis encrypt ...` commands here? You already have the DOCKER_PASS example below, so maybe you just meant to delete these two instructions.